crates/uv-cli/src/lib.rs

```diff
@@ -2360,9 +2360,18 @@ pub struct LockArgs {
 #[derive(Args)]
 #[allow(clippy::struct_excessive_bools)]
 pub struct AddArgs {
-    /// The packages to add, as PEP 508 requirements (e.g., `ruff==0.5.0`).
-    #[arg(required = true)]
-    pub requirements: Vec<String>,
+    /// Install all listed packages.
```

Comment:
Not quite "Install" here, can we retain the previous description? It'd be good to move the "as PEP 508" part to a second line though so it's only present in the long help.